### ISSUE TITLE
Responsive Design

### ISSUE THREAD

**Issue by Abbe98** [wikimaps-dev/wikimaps-html#5]
We also has to decide on what should be available through mobile/tablets, this can't be a mobile first tool set?

**Comment by Abbe98** [comment]
I need input what to include on smaller screens @susannaanas.

**Comment by susannaanas** [comment]
I have to go through it, don't know yet. Everything could be different or
then we can make everything usable on mobile.

I would like a mobile georeferencing app, which could well be a side
product of this.

Choose a map from a Commons category that fits your location. Walk to a
location shown on the map. Click a big blue button to declare I'm here.

maanantai 6. heinäkuuta 2015 Albin Larsson notifications@github.com
kirjoitti:
